zulipterminal/ui_tools/messages.py

```diff
@@ -723,14 +723,24 @@ def main_view(self) -> List[Any]:
         if self.message["id"] in self.model.index["edited_messages"]:
             edited_label_size = 7
             left_padding = 1
+            is_edited = True
```

Comment:
Instead of ``is_edited = True``, you can replace it directly with ``label_text = "EDITED`` and remove last conditional to reduce no. of code lines while keeping the code logic intact.
Same can be done with ``is_edited = False``

Reply:
The current changes don't look like they've resolved this issue yet?

I'd also suggest moving this change into its own commit.